runtime/dotnet/azurewebapp/Scripts/deploy.ps1

```diff
@@ -115,49 +124,40 @@ if ($luisAuthoringKey -and $luisAuthoringRegion) {
 		"defaultLanguage" = $language;
 		"models"          = $noneEmptyModels
 	}
-	
+
 	$luString = $noneEmptyModels | Out-String
 	Write-Host $luString
 
 	$luconfigjson | ConvertTo-Json -Depth 100 | Out-File $(Join-Path $remoteBotPath luconfig.json)
 
 	# Execute bf luis:build command
 	if (Get-Command bf -errorAction SilentlyContinue) {
-		$customizedSettings = Get-Content $(Join-Path $remoteBotPath settings appsettings.json) | ConvertFrom-Json
-		$customizedEnv = $customizedSettings.luis.environment
-		
+
 		# create generated folder if not exists
 		if (!(Test-Path generated)) {
 			New-Item -ItemType Directory -Force -Path generated
 		}
-		
-		bf luis:build --luConfig $(Join-Path $remoteBotPath luconfig.json) --botName $name --authoringKey $luisAuthoringKey --dialog --out .\generated --suffix $customizedEnv -f --region $luisAuthoringRegion
```

Comment:
the argument --dialog without value causes an error. In the latest branch it mus be crosstrained